packages/ui/src/components/Menu/Item.tsx

```diff
@@ -81,9 +81,6 @@ const StyledLinkItem = styled('a', {
 type ItemProps = {
   href?: string | undefined
   disabled?: boolean | undefined
-  /**
-   * @deprecated : wrap Menu.Item with a Tooltip
-   */
```

Comment:
Why remove the deprecation notice ? 🤔

Reply:
It was ad because it was planned to depreciate this prop on long term but we changed our mind about that

Reply:
Alright :)